include/RAJA/util/View.hpp

```diff
@@ -162,6 +162,11 @@ struct MultiView
   layout_type const layout;
   nc_pointer_type data;
 
+  RAJA_INLINE constexpr MultiView()
+      : layout(layout_type()),
+        data(nullptr)
+  {}
```

Comment:
Why not put the initialization on the member declarations and use = default here?